### PR TITLE
refactor: migrate welcome-app

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Frontend: https://1219-pr-launcher-app-launcher-frontend-fabric8-launcher.surge.sh
- Storybook: https://1219-pr-storybook-launcher-frontend-fabric8-launcher.surge.sh

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Frontend: https://1219-pr-launcher-app-launcher-frontend-fabric8-launcher.surge.sh
- Welcome App: https://1219-pr-launcher-welcome-app-launcher-frontend-fabric8-launcher.surge.sh- Storybook: https://1219-pr-storybook-launcher-frontend-fabric8-launcher.surge.sh

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Frontend: https://1219-pr-launcher-app-launcher-frontend-fabric8-launcher.surge.sh
- Welcome App: https://1219-pr-launcher-welcome-app-launcher-frontend-fabric8-launcher.surge.sh
- Storybook: https://1219-pr-storybook-launcher-frontend-fabric8-launcher.surge.sh

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Frontend: https://1219-pr-launcher-app-launcher-frontend-fabric8-launcher.surge.sh
- Welcome App: https://1219-pr-launcher-welcome-app-launcher-frontend-fabric8-launcher.surge.sh
- Storybook: https://1219-pr-storybook-launcher-frontend-fabric8-launcher.surge.sh

**Comment by ia3andy** [comment]
This is just a starting point:
- It still needs some refactor
- I commented the tests out (they need to be remade)

File: packages/launcher-welcome-app/.circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Javascript Node CircleCI 2.0 configuration file
```
Comment (gastaldi) [inline packages/launcher-welcome-app/.circleci/config.yml]: Is this file necessary?

File: packages/launcher-welcome-app/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+
+# Launcher Creator Welcome App
+[![Build Status](https://travis-ci.org/fabric8-launcher/launcher-creator-welcome-app.svg?branch=master)](https://travis-ci.org/fabric8-launcher/launcher-creator-welcome-app)
```
Comment (gastaldi) [inline packages/launcher-welcome-app/README.md]: This badge needs to be updated (or removed)
